src/com/intellij/idea/plugin/hybris/tools/remote/execution/logging/LoggingExecutionClient.kt

```diff
@@ -43,6 +43,8 @@ import java.nio.charset.StandardCharsets
 @Service(Service.Level.PROJECT)
 class LoggingExecutionClient(project: Project, coroutineScope: CoroutineScope) : ExecutionClient<LoggingExecutionContext, LoggingExecutionResult>(project, coroutineScope) {
 
+    val cxLoggerUtilities: CxLoggerUtilities = CxLoggerUtilities.getInstance(project)
```

Comment:
move it to method or make private, I do not remember is it ok to cache services.